README.md

```diff
@@ -155,6 +155,12 @@ Default: `250`
 Sets the delay in milliseconds after typing before a request will be sent to find suggestions.
 Specify `0` if you wish to fetch suggestions after every keystroke.
 
+#### highLightMatch
```

Comment:
Name it `highlightMatch` as highlight is one word (and we are not talking about high lights 😉).